test/example_data.dart

```diff
@@ -388,6 +388,16 @@ const _unreadMsgs = unreadMsgs;
 // Events.
 //
 
+DeleteMessageEvent deleteMessageEvent(List<StreamMessage> messages) {
+  return DeleteMessageEvent(
+    id: 0,
+    messageIds: messages.map((message) => message.id).toList(),
+    messageType: MessageType.stream,
+    streamId: messages[0].streamId,
+    topic: messages[0].topic,
+  );
+}
```

Comment:
Could be good to assert that all the messages have the same stream ID and topic. The [API doc](https://zulip.com/api/get-events#delete_message) isn't totally clear, but it seems like an imagined event with messages in different streams/topics wouldn't be representative. Maybe like this:

```suggestion
DeleteMessageEvent deleteMessageEvent(List<StreamMessage> messages) {
  assert(messages.isNotEmpty);
  final streamId = messages.first.streamId;
  final topic = messages.first.topic;
  assert(messages.every((m) => m.streamId == streamId));
  assert(messages.every((m) => m.topic == topic));
  return DeleteMessageEvent(
    id: 0,
    messageIds: messages.map((message) => message.id).toList(),
    messageType: MessageType.stream,
    streamId: streamId,
    topic: topic,
  );
}
```